src/stories/3-fields/slider/slider.stories.tsx

```diff
@@ -41,10 +41,10 @@ const meta: Meta = {
 		},
 		customOptions: {
 			description:
-				"<ul><li>`showSliderLabels` specifies if max and min labels are displayed.</li><br/><li>Use `sliderLabelPrefix` and `sliderLabelSuffix` to customise the labels</li></ul>",
+				"<ul><li>`showSliderLabels` specifies if max and min labels are displayed.</li><br/><li>Use `sliderLabelPrefix` and `sliderLabelSuffix` to customise the labels</li><br /> <li> `showIndicatorLabel` specifies if indicator will be displayed </li> <br/> <li> Use `indicatorLabelPrefix` and `indicatorLabelSuffix` to customise the the indicator. </li> <br/> </ul>",
```

Comment:
small typo, extra `the` set for the last line

```suggestion
				"<ul><li>`showSliderLabels` specifies if max and min labels are displayed.</li><br/><li>Use `sliderLabelPrefix` and `sliderLabelSuffix` to customise the labels</li><br /> <li> `showIndicatorLabel` specifies if indicator will be displayed </li> <br/> <li> Use `indicatorLabelPrefix` and `indicatorLabelSuffix` to customise the indicator. </li> <br/> </ul>",
```

Reply:
Noted, removed the extra the